mcs/class/corlib/Test/System.Reflection/FieldInfoTest.cs

```diff
@@ -1356,6 +1356,7 @@ struct TestFields {
 		}
 
 		[Test]
+		[Category ("NotWorking")] // https://github.com/mono/mono/issues/18364
```

Comment:
looks like #18364 was fixed, can you try reenabling the test?

Reply:
oh, indeed. Fixed.